pkg/server/router/submission.go

```diff
@@ -0,0 +1,125 @@
+package router
+
+import (
+	"context"
+	"fmt"
+	"github.com/TBD54566975/ssi-sdk/credential/exchange"
+	"github.com/pkg/errors"
+	"github.com/tbd54566975/ssi-service/internal/keyaccess"
+	"github.com/tbd54566975/ssi-service/pkg/server/framework"
+	svcframework "github.com/tbd54566975/ssi-service/pkg/service/framework"
+	"github.com/tbd54566975/ssi-service/pkg/service/submission"
+	"net/http"
+)
+
+type SubmissionRouter struct {
+	service *submission.Service
+}
+
+func NewSubmissionRouter(s svcframework.Service) (*SubmissionRouter, error) {
+	if s == nil {
+		return nil, errors.New("service cannot be nil")
+	}
+	service, ok := s.(*submission.Service)
+	if !ok {
+		return nil, fmt.Errorf("could not create submission router with service type: %s", s.Type())
+	}
+	return &SubmissionRouter{service: service}, nil
+}
+
+type CreateSubmissionRequest struct {
+	PresentationJwt keyaccess.JWT `json:"presentationJwt" validate:"required"`
```

Comment:
nit: `PresentationJWT`

question: this better as `SubmissionJWT`?

Reply:
Used `SubmissionJWT`